common.go

```diff
@@ -434,3 +441,20 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 
 	return
 }
+
+// GetLanguagesByGitAttributes returns the language assigned in .gitattributes if the regular expresion
+// matchs with the filename
```

Comment:
Misleading comment: It returns a slice of languages, not a language.

The difference between this method and `GetLanguageByGitAttributes` should be made much clear and probably their names should be much different also.